tests/layer_tests/tensorflow_tests/test_tf_ArgMinMax.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from common.tf_layer_test_class import CommonTFLayerTest
+
+# Testing operation ArgMin, ArgMax (Initial Implementation)
+# Documentation: https://www.tensorflow.org/api_docs/python/tf/raw_ops/ArgMin
+#                https://www.tensorflow.org/api_docs/python/tf/raw_ops/ArgMax
+
+class TestArgMinMax(CommonTFLayerTest):
+    # input_shape - should be an array
+    # dimension - dimension to be used, for vector should be 0
+    # op_type - type of testing operation
+    # ir_version - common parameter
+    # use_new_frontend - common parameter
```

Comment:
please remove it. It looks obvious

Reply:
For you, probably, it is obvious. For someone who will see it first time in the future - I'm not sure. I suggest to keep.